src/main/java/edu/kis/powp/jobs2d/command/ComprisingCommandsCounterVisitor.java

```diff
@@ -0,0 +1,54 @@
+package edu.kis.powp.jobs2d.command;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class ComprisingCommandsCounterVisitor implements CommandVisitorInterface
+{
+    private Job2dDriver job2dDriver;
+    private int operateToCounter = 0;
+    private int setPositionCounter = 0;
+    private int CompoundCounter = 0;
```

Comment:
start name from the lowercase. I think that this field is not handled correctly (always 0)